### ISSUE TITLE
Archive this repo

### ISSUE THREAD

**Issue by b4handjr** [mozilla-services/ap-loadtester#128]
With the new loadtests ready to be added to ```mozilla-services```, we should archive this repo.

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/archiving-a-github-repository

**Comment by jrconlin** [comment]
Agreed. Once we have the link, I'll update the README to point to it and archive this repo.

Thanks!
